### DISCUSSION TITLE
Issue with Community License for Development

### DISCUSSION THREAD

**Discussion by TheGreatAxios** [ravendb/ravendb#19652] [Q&A] [answered]
Hey team -- 

I'm attempting to use Time Series rollups in my self hosted local testing environment using Docker because it's a community license when I pay you for multiple different production licenses and instances. Is there a way to use my cloud license locally so that I can actually test and build using these features? 

Overall -- love Raven, I've been a customer for many years; but feels like every feature is now blocked behind an extra paywall or license.

Thank you

**Comment by DejanMilicic** [comment, marked as answer]
You can use a Developer license for local development. It is pretty much unrestricted

Request it here - https://ravendb.net/license/request/dev

**Comment by TheGreatAxios** [comment]
Hey @DejanMilicic -- been a long time, hope everything is going well!

Thank you!
